snuba/stateful_consumer/states/bootstrap.py

```diff
@@ -14,12 +22,60 @@ class BootstrapState(State[ConsumerStateCompletionEvent, ConsumerStateData]):
     control topic.
     """
 
+    def __init__(self,
+        topic: str,
+        bootstrap_servers: Sequence[str],
+        group_id: str,
+    ):
+        super(BootstrapState, self).__init__()
+
+        def on_partitions_assigned(
+            consumer: Consumer,
+            partitions: Sequence[TopicPartition],
+        ):
+            pass
+
+        def on_partitions_revoked(
+            consumer: Consumer,
+            partitions: Sequence[TopicPartition],
+        ):
+            pass
+
+        self.__consumer = StrictConsumer(
+            topic=topic,
+            bootstrap_servers=bootstrap_servers,
+            group_id=group_id,
+            auto_offset_reset="earliest",
+            partition_assignment_timeout=settings.SNAPSHOT_CONTROL_TOPIC_INIT_TIMEOUT,
+            on_partitions_assigned=on_partitions_assigned,
+            on_partitions_revoked=on_partitions_revoked,
+            on_message=self.__handle_msg,
+        )
+
+    def __handle_msg(self, message: Message) -> CommitDecision:
+        logger.info(
+            "MSG %r %r %r",
+            message,
+            message.value(),
+            message.error(),
+        )
+        # TODO: Actually do something with the messages and drive the
+        # state machine to the next state.
+        return CommitDecision.DO_NOT_COMMIT
+
     def signal_shutdown(self) -> None:
         pass
 
     def handle(self, state_data: ConsumerStateData) -> Tuple[ConsumerStateCompletionEvent, ConsumerStateData]:
-        # TODO: Actually do the snapshot bootstrap
+        logger.info("Running Consumer")
```

Comment:
Might make sense to use the `%r` for `self.__consumer` here instead of just "Consumer".